examples/nextjs-lit/src/wrapped-simple-greeter.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import React from 'react';
+import {createComponent} from '@lit-labs/react';
+import {SimpleGreeter} from './simple-greeter';
+
+export default createComponent({
```

Comment:
"wrapped" is a bit of a generic name for me. I would name the module `simple-greeter-react` and at the import site just import it as `SimpleGreeter` (or maybe `SimpleGreeterReact` if you want to make it more clear the difference between it and `<simple-greeter>`).

Reply:
sounds good, done.